src/tests/all.rs

```diff
@@ -865,76 +855,69 @@ impl MockUserSession {
         f(&conn)
     }
 
-    /// For internal use only: make the current request
```

Comment:
I wasn't totally happy with this API... and you've made it so much better. I don't know why I didn't consider just adding the authentication info to every request, but it makes so much sense now that you've implemented it!